qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -133,12 +133,11 @@
         - @default:
         - data_plane:
             no Host_RHEL.m6
-            only with_plug.with_system_reset.one_pci
-            one_pci:
-                repeat_times = 1
+            only with_plug.with_system_reset.one_pci, block_scsi..with_plug.with_repetition.one_pci
             iothreads = iothread0
             block_virtio:
                 blk_extra_params_stg0 = "iothread=${iothreads}"
             block_scsi:
                 no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
+                bus_extra_params_image1 = "iothread=${iothreads}"
```

Comment:
useless param setting for driveformat=virtio_blk; suggest setting it only for driveformat=virtio_scsi.
So, my suggestion is:
   block_scsi:
       no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
       bus_extra_params_stg0 = "iothread=${iothreads}"
       virtio_scsi:
           bus_extra_params_image1 = "iothread=${iothreads}"

Or any other methods you can take to avoid the redundant param setting issue.

Reply:
bus_extra_params_image1 is just for virtio_scsi. I think no need modify it. When system is blk, it has no effect.

Reply:
@luckyh , @CongLi 
  Test result of this patch is "PASS".  If you think the issue i commented is ok, i'll give my ack on this patch.

Reply:
'bus_extra_params' is defined for virtio-scsi-pci bus only, I think it's fine.        
:param bus_extra_params: options want to add to virtio-scsi-pci bus

Thanks.

Reply:
ACK